spec/models/mixins/ownership_mixin_spec.rb

```diff
@@ -0,0 +1,26 @@
+describe "Ownership" do
+  describe "reporting" do
+    describe "searching by current ldap group" do
+      let(:exp)    { {"="=>{"field"=>"Vm-owned_by_current_ldap_group", "value"=>"true"}} }
+      let(:report) { MiqReport.new.tap {|r| r.db = 'Vm'} }
+      let(:search_opts) { { :filter => MiqExpression.new(exp), :per_page => 20, :named_scope => :with_miq_group } }
+
+      before do
+        @miq_group = FactoryGirl.create :miq_group
+        User.current_user = FactoryGirl.create :user,
+                                               :current_group => @miq_group,
+                                               :miq_groups => [@miq_group]
+        expect(User).to receive(:server_timezone).and_return("UTC")
```

Comment:
Few things:

* It is `login_as`, not `login_as_user`
* Not going to use `stub_user` because backwards compatible, and I would like this to go into `darga`.
* Logging in the user also need to happen **AFTER** the groups are created.  Found that out when I switched to @Fryguy 's [suggestion](https://github.com/ManageIQ/manageiq/pull/10290#discussion_r73751496) and the specs failed (a better way to test by the way since I had a false positive passing test).  `set_ownership` (I think) is called somewhere and assigns it to the `current_user`'s group on create (lost about 2 hours on this one).
* Also, unless this is a [view](https://github.com/ManageIQ/manageiq/blob/master/spec/spec_helper.rb#L48), [controller](https://github.com/ManageIQ/manageiq/blob/master/spec/spec_helper.rb#L54), or [helper](https://github.com/ManageIQ/manageiq/blob/master/spec/spec_helper.rb#L73) spec, both `stub_user` and `login_as` methods are not available in a model test.

Which raise another question, is there a better place for this test?  I really didn't want to do this in a controller test, but I guess I could.  Just seemed like it was a bit verbose to test this with a full action and rendering when I just wanted to know the results from the filter.

Thoughts?